chains/sei-evm-testnet.json

```diff
@@ -0,0 +1,28 @@
+{
+  "alias": "sei-evm-testnet",
+  "blockTimeMs": 2951,
+  "decimals": 18,
+  "explorer": {
+    "api": {
+      "key": {
+        "required": false
+      },
+      "url": "https://seitrace.com/api"
+    },
+    "browserUrl": "https://seitrace.com/"
+  },
+  "id": "713715",
+  "name": "Sei EVM testnet",
```

Comment:
There is also testnet for this chain which code name is `atlantic-2`. So it's better to be explicit and say `Sei EVM devnet`.